app/components/UserValidator/index.tsx

```diff
@@ -127,6 +132,17 @@ const Validator = (props: Props) => {
             facingMode: 'environment',
           }}
         />
+        <h3>Nylig scannede ABA-IDer</h3>
+        <Flex column justifyContent="center" gap={5}>
+          {scanResults.map(({ message, count }) => (
+            <span key={message + count}>
+              {message} (x{count})
+            </span>
+          ))}
```

Comment:
Am I reading the code wrong, or won't this list just keep growing? Won't that look weird?

Reply:
You're reading it absolutely right. Personal preference I guess - and I don't really mind doing it either way - but I tested it by generating an overflow of messages and it looked a-ok to me.

The behaviour currently is that the modal grows in height after scans - but as it approaches the max height of the screen (the maxheight attribute is set to leave some space on the top and bottom - which is nice), the oldest results at the bottom overflow and are not seen unless the user scrolls down to view them.  
If we dont want it to resize I could set a maxHeight to the Flex div that displays the result

Reply:
Updated it now to only display the 5 last results